src/api/backend.js

```diff
@@ -25,32 +29,48 @@ export const getUsersByAddress = async (blockchainAddress) => {
   return data.map(({ uid, merits }) => ({
     uid,
     merits: ethers.utils.parseUnits(merits.toFixed(12), 12),
-  }))
+  }));
+};
+
+export const maybeGetApprovedEresidency = async () => {
+  const api = getApi();
+  try {
+    const approvedEresidency = await api.get('/e-residents/approved/me');
+    return approvedEresidency.data;
+  } catch (e) {
+    return false;
```

Comment:
Hmm... so maybe to clarify - if user is not approved, will we return the `data` or will we return false?

Reply:
Yea, so that's bad. If API fails for whatever reason (503 or connection issues) we'll return false and tell the clerk that the user is not approved. The clerk then will go to the user and say that they need to get approved first. But the user will see that he is approved and everyone will be confused.
Maybe add a special handling for just `404` status code to make sure we're not mistaking it with other problems?

Reply:
Theres a different message now for 401 and 404, versus other response, versus other error